firebase-firestore/src/main/java/com/google/firebase/firestore/model/mutation/Mutation.java

```diff
@@ -93,33 +93,25 @@ public Precondition getPrecondition() {
   }
 
   /**
-   * Applies this mutation to the given MaybeDocument for the purposes of computing a new remote
-   * document. If the input document doesn't match the expected state (e.g. it is null or outdated),
-   * an `UnknownDocument` can be returned.
+   * Applies this mutation to the given Document for the purposes of computing a new remote document
+   * If the input document doesn't match the expected state (e.g. it is null or outdated), an
+   * `UnknownDocument` may be created
```

Comment:
"may be created" is somewhat misleading, since the `MutableDocument` is modified in place. Please update.